New major – Support for additional types (#67)
sanity-io/assist: plugin/README.md

}
```

### Assist configuration (optional)

```ts
assist({

>   //Showing defaults

Reviewer: ```suggestion
  // Showing default values
```
Mostly just needs the space after // for consistency with other comments

  assist: {
    localeSettings: () => Intl.DateTimeFormat().resolvedOptions(),
    maxPathDepth: 4,
    temperature: 0.3
  },
  translate: { /* see sections about document and field translation */}
})
```

- `localeSettings`: See section on [date and datetime](#date-and-datetime)
- `maxPathDepth`: The max depth for document paths AI Assist will write to.
- `temperature`: Influences how much the output of an instruction will vary between runs.

> For more details, please confer the TSDocs of the individual config parameters in [assistTypes.ts](./src/assistTypes.ts)

Reviewer: ```suggestion
For more details, please review the TSDocs of the individual config parameters in [assistTypes.ts](./src/assistTypes.ts)
```

Objects where all fields are excluded or unsupported and arrays where all member types are excluded or unsupported
will also be excluded.

### Date and datetime
- [Date](https://www.sanity.io/docs/date-type)
- [Datetime](https://www.sanity.io/docs/datetime-type)

Starting from v3.0.0, AI Assist can write to date and datetime fields. Instructions can use language like "tomorrow at noon" or
"next year", and when Assist writes to the field, it will be converted to a field-compatible value.

Language about time is locale and timeZone dependant. By default instructions will use the locale and timezone provided

> by the browser (`Intl.DateTimeFormat().resolvedOptions()`)

Reviewer: ```suggestion
by the browser (`Intl.DateTimeFormat().resolvedOptions()`).
```

> Alternatively, the plugin can configured per user with a `assist.localeSettings` function that should return `LocaleSettings`.

Reviewer: ```suggestion
Alternatively, you can configure the plugin per user with an `assist.localeSettings` function that should return `LocaleSettings`.
```

##### Example
```ts
assist({
  assist: {
    localeSettings: ({currentUser, defaultSettings}) => {
      if (currentUser.roles.includes('admin')) {

>         //forces locale and timeZone for admins

Reviewer: ```suggestion
        // forces locale and timeZone for admins
```
